pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1437,7 +1437,7 @@ func TestPodEligibleToPreemptOthers(t *testing.T) {
 			pods:                []*v1.Pod{st.MakePod().Name("p1").UID("p1").Priority(lowPriority).Node("node1").Terminating().Obj()},
 			nodes:               []string{"node1"},
 			nominatedNodeStatus: nil,
-			expected:            false,
+			expected:            true,
```

Comment:
why is this changing?

I suppose this is now covered by "non-victim Pods terminating"

Reply:
> why is this changing?

because I started to default `test.fts.EnablePodDisruptionConditions` to `true`

> I suppose this is now covered by "non-victim Pods terminating"

Correct, I drop the redundant test case.